### PR TITLE
Enhance/chat

### PR DESCRIPTION
**Pull request by pinocchio-life-like**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
   "packageManager": "yarn@4.1.1",
   "dependencies": {
     "@babel/runtime": "^7.21.0",
+    "@instructor-ai/instructor": "^1.4.0",
```
Comment (andrew-bierman) [inline package.json:91]: we can add this to server directly, not needed in root

**Review by andrew-bierman — APPROVED** [review]
nice job on this, added some clean up tasks but great work :)

File: package.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     "next": "cd apps/next && yarn dev",
     "prebuild:expo": "cd apps/expo && yarn run prebuild:clean",
     "postinstall": "yarn build-ui && yarn build:validations && yarn build-crosspath && yarn setup:env && yarn fix:packages",
-    "preinstall:yarn": "npx only-allow yarn",
```
Comment (andrew-bierman) [inline package.json:58]: Revert

File: package.json [hunk 3]
```diff
@@ -96,10 +96,13 @@
     "eslint": "^8.56.0",
     "husky": "^8.0.0",
     "lint-staged": "^14.0.1",
+    "lucide-react-native": "^0.396.0",
     "npm-run-all": "^4.1.5",
     "patch-package": "^8.0.0",
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^3.2.5",
+    "react-native-reanimated": "^3.12.0",
+    "react-native-safe-area-context": "^4.10.4",
```
Comment (andrew-bierman) [inline package.json:105]: Add to /app directory instead of root

**Review by andrew-bierman — APPROVED** [review]

**Comment by pinocchio-life-like** [comment]
@andrew-bierman  I have done the cleanup and this should be good to merge.

**Review by andrew-bierman — APPROVED** [review]

**Pull request merged**